src/example.py

```diff
@@ -1,9 +1,9 @@
-from spireslayer.templates.defect_deck import frost_deck
+from spireslayer.templates.watcher_deck import *
 from spireslayer.save_editor import SaveEditor
 
 
 if __name__ == '__main__':
-    save_file_path = "C:\\Program Files (x86)\\Steam\\steamapps\\common\\SlayTheSpire\\saves"
+    save_file_path = "/path/to/save/folder"
```

Comment:
Same opinion as the `save_editor.py`, plus this is an example script that people will likely copy from. So I think the original code will informs better about how the path should be written.

With `/path/to/save/folder`, we may need to add additional documentation inside `readme.md` to "please point it out to the `saves` folder inside a game root".

Reply:
Agree ! (Same as other review)